cmd/merge-nodes/internal/merge_action_test.go

```diff
@@ -227,13 +228,13 @@ func Test_MergeDBs_Successful_Existing_Node(t *testing.T) {
 	require.NoError(t, err)
 
 	sig1Poet1 := nipost.PoETRegistration{
-		ChallengeHash: sig1Ch.Hash(),
+		ChallengeHash: wire.NIPostChallengeToWireV1(sig1Ch).Hash(),
```

Comment:
I think you can use a random hash here instead of having to use the wire type in the test (`merge-nodes` shouldn't care about wire types at all).